tests/trx_generator/main.cpp

```diff
@@ -48,6 +52,9 @@ int main(int argc, char** argv) {
          ("trx-gen-duration", bpo::value<uint32_t>(&gen_duration)->default_value(60), "Transaction generation duration (seconds). Defaults to 60 seconds.")
          ("target-tps", bpo::value<uint32_t>(&target_tps)->default_value(1), "Target transactions per second to generate/send. Defaults to 1 transaction per second.")
          ("last-irreversible-block-id", bpo::value<string>(&lib_id_str), "Current last-irreversible-block-id (LIB ID) to use for transactions.")
+         ("monitor-spinup-time-us", bpo::value<int64_t>(&spinup_time_us)->default_value(1000000), "Number of microseconds to wait before monitoring TPS. Defaults to 1000000 (1s).")
+         ("monitor-max-lag-percent", bpo::value<uint32_t>(&max_lag_per)->default_value(0), "Max percentage off from expected transactions sent before being in violation. Defaults to 5.")
```

Comment:
Says defaults to 5 but default value is 0.

Reply:
fixed